### PR TITLE
Adds documentation for new integration flexit_bacnet

### PR DESCRIPTION
**Pull request by lellky**
## Proposed change

Adds documentation for new integration flexit_bacnet

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104275
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4923
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/flexit_bacnet.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+Integrates [Flexit](https://www.flexit.no/en/) Nordic series air handling unit into Home Assistant.
+
+## Prerequisites
+
+To configure the integration, you need to obtain IP address and Device ID for the unit.
+
+1. Open Flexit Go app on your mobile.
+2. Use "Find product" button on tha main screen.
```
Comment (c0ffeeca7) [inline source/_integrations/flexit_bacnet.markdown]:
```suggestion
2. On the main screen, select the **Find product** button.
```

File: source/_integrations/flexit_bacnet.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ha_integration_type: integration
+---
+
+Integrates [Flexit](https://www.flexit.no/en/) Nordic series air handling unit into Home Assistant.
+
+## Prerequisites
+
+To configure the integration, you need to obtain IP address and Device ID for the unit.
+
+1. Open Flexit Go app on your mobile.
```
Comment (c0ffeeca7) [inline source/_integrations/flexit_bacnet.markdown]:
```suggestion
1. Open the Flexit Go app on your mobile.
```

File: source/_integrations/flexit_bacnet.markdown [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+2. Use "Find product" button on tha main screen.
+3. Select your device and press "Connect".
```
Comment (c0ffeeca7) [inline source/_integrations/flexit_bacnet.markdown]:
```suggestion
3. Select your device and select **Connect**.
```

File: source/_integrations/flexit_bacnet.markdown [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+4. Enter installer code (default: 1000) and press "Login".
```
Comment (c0ffeeca7) [inline source/_integrations/flexit_bacnet.markdown]:
```suggestion
4. Enter the installer code (default: 1000) and select **Login**.
```

File: source/_integrations/flexit_bacnet.markdown [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+5. Open "More" menu -> Installer -> Communication -> BACnet settings.
```
Comment (c0ffeeca7) [inline source/_integrations/flexit_bacnet.markdown]:
```suggestion
5. Go to **More** > **Installer** > **Communication**  > **BACnet settings**.
```

File: source/_integrations/flexit_bacnet.markdown [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+6. Note down "IP address" and "Device ID".
```
Comment (c0ffeeca7) [inline source/_integrations/flexit_bacnet.markdown]:
```suggestion
6. Note down the **IP address** and **Device ID**.
```

**Review by c0ffeeca7 — COMMENTED** [review]
Tweaks. In the docs, **bold** is used to markup UI strings.

File: source/_integrations/flexit_bacnet.markdown [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+Integrates [Flexit](https://www.flexit.no/en/) Nordic series air handling unit into Home Assistant.
+
+## Prerequisites
+
+To configure the integration, you need to obtain IP address and Device ID for the unit.
```
Comment (c0ffeeca7) [inline source/_integrations/flexit_bacnet.markdown]:
```suggestion
To configure the integration, you need to obtain the IP address and Device ID for the unit.
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @lellky :+1:

File: source/_integrations/flexit_bacnet.markdown [hunk 8]
```diff
@@ -0,0 +1,27 @@
+---
+title: Flexit BACnet
+description: Instructions on how to integrate Flexit air handling unit into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2023.12
```
Comment (joostlek) [inline source/_integrations/flexit_bacnet.markdown]:
```suggestion
ha_release: 2024.1
```

**Review by joostlek — APPROVED** [review]

**Pull request merged**